test/benchmarks/driver_bench/src/main.mts

```diff
@@ -198,6 +199,31 @@ function calculateCompositeBenchmarks(results: MetricInfo[]) {
   return [...results, ...compositeResults];
 }
 
-const finalResults = calculateCompositeBenchmarks(results);
+function calculateNormalizedResults(results: MetricInfo[]): MetricInfo[] {
+  const baselineBench = results.find(r => r.info.test_name === 'cpuBaseline');
+  const pingBench = results.find(r => r.info.test_name === 'ping');
 
-await fs.writeFile('results.json', JSON.stringify(finalResults, undefined, 2), 'utf8');
+  assert.ok(pingBench, 'ping bench results not found!');
+  assert.ok(baselineBench, 'baseline results not found!');
+  const pingThroughput = pingBench.metrics[0].value;
+  const cpuBaseline = baselineBench.metrics[0].value;
+
+  for (const bench of results) {
+    if (bench.info.test_name === 'cpuBaseline') continue;
+    if (bench.info.test_name === 'ping') {
+      bench.metrics.push({ 'name': 'normalized_throughput', value: bench.metrics[0].value / cpuBaseline });
```

Comment:
I can tell prettier isn't running here, which means we're not linting here, if the eslint config doesn't present an obvious way to fix that can we at least run prettier manually before merging.

Reply:
Oh! we just need `,.mts'` in the `--ext '.js,.ts,.mts'` flag passed to eslint!

There's a handful of import sort fixes, plus the prettier, no errors otherwise 😅

Reply:
Got it, will fix that up, but it looks like there would also be changes to our aws lambda test file, so going to leave the check:eslint script as is for now and put up a separate PR to address that.

Reply:
Sure, but can we still push up the changes that go on these files?

Reply:
Yep yep!